README.md

```diff
@@ -1,2 +1,181 @@
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+- **ORM**
+- **ROR**
+- **Linters usage**
+- **Test with Rspec**
+- **Test with Capybara**
+- **Authorization**
+- **Authentication**
+- **MVC**
+- **Deploy**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Git SCM
+- Ruby installed
+- Rails installed
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone https://github.com/NelsonAraujo93/Budget-app.git
+```
+
+Or download the ZIP folder.
+
+### Install
+
+Once you've cloned the repository simply run:
+
+```
+bundle install
+rails db:create
+rails db:migrate
+```
+
+And this should open the directory with all the files from the repository on the command line, to open it in your explorer if you're using windows you can run:
+
+```
+explorer .
+```
+
+### Usage
+
+To run the project, if you are on the Recipe-app folder path on your terminal you can execute the following command:
+
+```
+code .
+```
+
+This will open Visual Studio Code on the folder you're at.
+
+open terminal and run
+
+```
+  rails server
+```
+
+### Run tests
+ To start tests run:
+```
+rspec spec
+```
+
+
+### Deployment
+
+Deployment link - [Live-demo](https://rails-budget-app-prxn.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Nelson Araujo**
+
+- GitHub: [@NelsonAraujo93](https://github.com/NelsonAraujo93)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/nelson-araujo-paredes/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+- **Make improvements with feedbacks**  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/NelsonAraujo93/Budget-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can just cheer me up with a comment 🙂!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank  [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) for the Original design idea on Behance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **What is this project main purpose?**
+
+  - To learn Ruby on Rails
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please fix the license link on your README file the current license link is broken.